src/common/common/file_io.cpp

```diff
@@ -15,3 +15,9 @@ std::string loadFileContents(const std::string& path)
     stream << inFile.rdbuf();
     return stream.str();
 }
+
+bool fileExists(const std::string& path)
+{
+    std::ifstream inFile(path);
+	return !!inFile;
+}
```

Comment:
C++17 has `<filesystem>`, which allows you to do `std::filesystem::exists(path)`

https://en.cppreference.com/w/cpp/filesystem/exists

So this function isn't necessarily needed (or it can be simple wrapper around the c++17 function, which looks kinda long)